src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CcdCaseUpdater.java

```diff
@@ -148,6 +157,28 @@ public ProcessResult updateCase(
             log.error(msg);
             ClientServiceErrorResponse errorResponse = new ClientServiceErrorResponse(asList(msg), emptyList());
             return new ProcessResult(errorResponse.warnings, errorResponse.errors);
+        } catch (FeignException exception) {
+            log.error(
+                "Failed to update case for {} service with case Id {} based on exception record {}. "
+                    + "Service response: {}",
+                configItem.getService(),
+                existingCaseId,
+                exceptionRecord.id,
+                exception.contentUTF8(),
+                exception
```

Comment:
why do we need both ?

Reply:
Are you sure we should log this and this exception won't have any sensitive data ?

Reply:
'exception.contentUTF8()' is a part of the logged message, 'exception' is the second parameter of 'log.error(...)' call

Reply:
As discussed before (with Leszek) the service itself should not send us sensitive data. Logging error responses will significantly help in the future to support the app